packages/@tailwindcss-postcss/src/__snapshots__/index.test.ts.snap

```diff
@@ -10,11 +10,7 @@ exports[`\`@import 'tailwindcss'\` is replaced with the generated CSS 1`] = `
     --text-2xl--line-height: calc(2 / 1.5);
     --font-weight-bold: 700;
     --default-font-family: var(--font-sans);
-    --default-font-feature-settings: var(--font-sans--font-feature-settings);
-    --default-font-variation-settings: var(--font-sans--font-variation-settings);
     --default-mono-font-family: var(--font-mono);
-    --default-mono-font-feature-settings: var(--font-mono--font-feature-settings);
-    --default-mono-font-variation-settings: var(--font-mono--font-variation-settings);
```

Comment:
Now that the `--default-font-feature-settings` theme variable is defined as:

```css
@theme {
  --default-font-feature-settings: --theme(--font-sans--font-feature-settings, initial);
}
```

The value is resolved to `initial` (and thus omitted) because we know that `--font-sans--font-feature-settings` is not defined in the theme. Note that this implies that values referenced via `--theme(…)` can not be defined at runtime only and NEED to be in the `@theme` block. I think this is a reasonable thing to do since we also _error_ when it's not in the `@theme` block (unless a fallback is specified like it is here).